Hero section Branch

KareemWilson opened the pull request:
I tried to do my best. 

In this project I : 

1. chose template number 3, in the link provided with the project [here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)
2. began with NavBar and the Welcome Section, and I am almost finished with them.
3. made a responsive layout by making a hamburger menu button appear on mobile view, and made a suitable view for mobiles.

emmiiorji commented on README.md:
```diff
@@ -1 +1,18 @@
+- CSS
+
+
+
+## Authors
+
+👤 **Kareem Magdy Wilson**
+
+- GitHub: [KareemWilson](https://github.com/KareemWilson)
+- LinkedIn: [Kareem Wilson](https://www.linkedin.com/in/kareem-wilson/)
+
```
Kindly include all the main headers of the given [README template](https://github.com/microverseinc/readme-template/blob/master/README.md). They'll make your documentation more professional.

Include the _Contributing_, _Show your support_, _Acknowledgments_, and _License_ sections of the template.

emmiiorji commented on index.html:
```diff
@@ -0,0 +1,45 @@
+  </head>
+  <body>
+    <div class="container">
+      <ul class="nav-list">
+        <li class="nav-item"><a href="#Welcome">Portofolio</a></li>
+        <li class="nav-item"><a href="#About">About</a></li>
+        <li class="nav-item"><a href="#Contact">Contact</a></li>
```
Kindly include a header tag which will be a container for the logo and the navigation list. Doing so makes for better semantics.

emmiiorji commented on index.html:
```diff
@@ -0,0 +1,45 @@
+        <li class="nav-item"><a href="#Welcome">Portofolio</a></li>
+        <li class="nav-item"><a href="#About">About</a></li>
+        <li class="nav-item"><a href="#Contact">Contact</a></li>
+        <li class="fa fa-bars"></li>
```
Please, use the hamburger button within a nav tag for better semantics.

emmiiorji commented on styles.css:
```diff
@@ -0,0 +1,140 @@
+  margin-top: 50px;
+}
+
+.fa {
+  justify-content: space-between;
+  margin-right: 15px;
+  margin-left: 15px;
+}
```
Kindly adjust the size of the hamburger button to match the design. Check the below comment for more on why you should stick to the given design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/185383912-4a6d377b-e4da-4175-a92e-e196ad953623.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/185383972-f47c9033-fd2c-439c-a8d8-da88225e4aec.png)

emmiiorji commented on styles.css:
```diff
@@ -0,0 +1,140 @@
+    text-align: start;
+  }
+
+  .bio {
+    text-align: start;
+    max-width: 100%;
+  }
+}
```
Kindly ensure that the alignment, font size and color of the main title and bio paragraph match that given in the design. Check the below comment for more on why you should stick to the given design.

You have a left alignment while the design has a center alignment.

Your design:
![image](https://user-images.githubusercontent.com/20554583/185386137-6a040f10-f7f7-437a-ab86-83b60d6032eb.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/185386255-473ce9cf-004f-4dbe-a4d1-dfff5762f5ba.png)

emmiiorji commented on styles.css:
```diff
@@ -0,0 +1,140 @@
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  background-image: url(./utils/Header-llustration-desktop@2x.png);
```
Kindly ensure that, following the design, the background image shows up properly. Not only does it look better, but you'll also get more prepared for a typical workplace experience where you might be required to reproduce a given design as strictly as possible. So, sticking to the design as much as possible (e.g. font, colors, images, text, margins) will be a good habit.

Follow the original vision if it's pretty and works, it will be a wonderful website.

Your design:
![image](https://user-images.githubusercontent.com/20554583/185387739-da0f66a3-badb-4ab5-8a90-166c268eaa0d.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/185387760-54618642-7cd6-4546-a847-129fc11945b7.png)

emmiiorji commented on index.html:
```diff
@@ -0,0 +1,45 @@
+        <section id="Welcome" class="welcome-section">
+          <h1 class="title">Hello there, I'm Kareem.</h1>
+          <h2 class="subTitle">I'm a Software Developer</h2>
+          <h4 class="bio">
+            Microuant in Microverse, My favourite hobby turning coffee and
+            legacy code into clean code, Looking forward to play with the chips
+            of Computers!
+          </h4>
```
Please, use a paragraph (`<p>`) tag for the bio. This way will be semantically correct and the screen readers and other technologies will thank you.

emmiiorji reviewed (changes requested):
Hi @KareemWilson, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![Just-a-little-bit](https://user-images.githubusercontent.com/20554583/185378864-3f5064a9-f0b4-4245-a05f-7a94e297c476.gif)


## Highlights

- Descriptive commit messages
- No linter errors


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

KareemWilson commented:
I'm very sorry, it was already many mistakes that are not overlooked.
this time I did more things which makes my portfolio prettier. 

In this Review I : 
1. Changed the Readme File. 
2. Used a header tag and nav tag.
3. changed alignments of all texts to be as same as the template.
4. adjusted the size of the hamburger button to match the template.
5. used `<p>` tag to show the bio 

After all, I really appreciate your efforts and help.

codepantha commented on README.md:
```diff
@@ -1 +1,53 @@
+- Hat tip to anyone whose code was used
+- Inspiration
+
+## 📝 License
+
+This project is [Kareem wilson](https://github.com/KareemWilson) licensed.
```
- [x] Well done for adding the License section 👏 However, kindly use the MIT license

codepantha reviewed (changes requested):
# STATUS: REQUIRED CHANGES ♻️ 

## Hi @KareemWilson 👋, 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

<img src="https://media0.giphy.com/media/is14HLE2EUHPryPkmU/giphy.gif"/>

## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ Used correct GitHub Flow

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

KareemWilson commented:
I changed the Licence in ReadMe File.
So this is the last request review I have. I hope there is nothing to change.

Thank you.

tresorsawasawa commented on README.md:
```diff
@@ -1 +1,53 @@
+- Hat tip to anyone whose code was used
+- Inspiration
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
- [ ] Good job making the change required by the previous reviewer. So, I noticed that the **MIT** license link is still broken(`[MIT](./LICENSE)`). To fix that issue, just create a new file named `LICENSE` in your root directory, and inside add the content from the **[MIT](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** file provided from the README template repo.

tresorsawasawa commented on README.md:
```diff
@@ -1 +1,53 @@
-# MyPortfolio
\ No newline at end of file
+# Portfolio
+
+> My Personal Portfolio
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+**Run HTML file**
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+**pass Linters Tests**
+
+### Deployment
+
```
- [ **OPTIONAL** ] I think it would be better if you update your README file to make it more professional by improving the project description, content in some sections(such as the **Getting Started** section,...), removing the empty and non-related sections,... To make it easy for you just take a look at this **[README Template](https://github.com/tresorsawasawa/MyPortfolio#readme)** then complete your README to be similar to it.

tresorsawasawa reviewed (approved):
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @KareemWilson 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

![nodding-yes](https://camo.githubusercontent.com/aced9b1ffa1374fee6c3bcb9689701088f26a09dc5e75eaa59a01fd38ccc26f2/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f766d74786e78766556556f64472f67697068792e676966)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tresorsawasawa commented:
## Notice:
Hi @KareemWilson 

**Please make sure you make the above changes before merging your pull request**

Happy coding :tada:
